tests/upgrade_v2_script/static/prometheus_whole_config.output.yaml

```diff
@@ -10,29 +10,29 @@ kube-prometheus-stack:
   commonLabels: {}
   kubeApiServer:
     serviceMonitor:
-      interval:
+      interval: ""
```

Comment:
Is an empty string acceptable as a value for interval?

Reply:
I don't know but what I do know is that those values are checked and conditionally applied in `kube-prometheus-stack` chart

```
    {{- if .Values.kubeControllerManager.serviceMonitor.interval }}
    interval: {{ .Values.kubeControllerManager.serviceMonitor.interval }}
    {{- end }}
```

https://github.com/prometheus-community/helm-charts/blob/c3f8b974f7511a772405e1fbf28dfe8483a9a04e/charts/kube-prometheus-stack/templates/exporters/kube-controller-manager/servicemonitor.yaml#L21-L23